www/templates/layouts/base.html

```diff
@@ -35,6 +35,7 @@
     <meta name="theme-color" content="#0f4266">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
Use of jQuery is detrimental to the performance of the site and we don't want it here.